CI/before_script.msvc.sh

```diff
@@ -415,17 +413,23 @@ fi
 	if [ -z $APPVEYOR ]; then
 		cd $DEPS_INSTALL
 
+        # Boost's installer is still based on ms-dos API that doesn't support larger than 260 char path names
+        # We work around this by installing to root of the current working drive and then move it to our deps
+        
 		BOOST_SDK="$(real_pwd)/Boost"
+		CWD_DRIVE_ROOT=`powershell -command '(get-location).Drive.Root'`  # get the current working drive's root
 
-		if [ -d Boost ] && grep "BOOST_VERSION 106100" Boost/boost/version.hpp > /dev/null; then
+		if [ -d Boost ] && grep "BOOST_VERSION 106700" Boost/boost/version.hpp > /dev/null; then
 			printf "Exists. "
 		elif [ -z $SKIP_EXTRACT ]; then
 			rm -rf Boost
-			"${DEPS}/boost-1.61.0-msvc${MSVC_YEAR}-win${BITS}.exe" //dir="$(echo $BOOST_SDK | sed s,/,\\\\,g)" //verysilent
+			"${DEPS}/boost-1.67.0-msvc${MSVC_YEAR}-win${BITS}.exe" //DIR="${CWD_DRIVE_ROOT}Boost_temp" //VERYSILENT //NORESTART //SUPPRESSMSGBOXES //LOG="boost_install.log"			
```

Comment:
Perhaps the `//VERYSILENT //SUPPRESSMSGBOXES` could be set only when `[ -n "$CI" ]` as well? That way they shouldn't hide errors or warnings from non-CI users of the script.

Reply:
The whole thing doesn't work (as a script) if we don't use //VERYSILENT, so that has to stay. The rest we can hide behind a switch.